_data/story.yml

```diff
@@ -0,0 +1,22 @@
+description: We have achieved amazing things together!
+
+events:
```

Comment:
All this need to be rewritten, I was not sure what should be in here. Suggestions? :smile: